### DISCUSSION TITLE
Inspiration

### DISCUSSION THREAD

**Discussion by ghost** [reset-button/ariadne_django#31] [Ideas]
Hey, I just wanted to drop a link to an old repo of mine in case you're be interested.

It contains some class based resolvers + mixins, such as:
- List
- Detail
- Connection
- CRUD

https://gitlab.com/nilsdet/ariadne-django

It's currently dead, because it's code is highly project specific, so we moved that code to the project itself.
But some code might be interesting for this repo here.

**Comment by maxmorlocke** [comment]
Thanks for sharing.  We have a lot of tightly coupled ariadne+django code that we're slowly shipping to here as we make it less project specific.  I'll spend some more time looking through this!
